info.json

```
@@ -5,7 +5,7 @@
   "author": "Atria",
   "contact": "tomas.chmelik2@gmail.com",
   "homepage": "https://github.com/FUE5BASE/FUE5Exporter",
-  "factorio_version": "1.1",
-  "dependencies": ["base >= 1.1"],
+  "factorio_version": "2.0.11",
```

chore: should specify only first two versions
```suggestion
  "factorio_version": "2.0",
```